package.json

```diff
@@ -14,9 +14,25 @@
     "jquery",
     "csrf"
   ],
+  "license": "ISC",
+  "main": "dist/jquery-csrf-token.js",
   "files": [
-    "index.js"
+    "dist/**"
   ],
-  "license": "ISC",
-  "main": "index.js"
+  "devDependencies": {
+    "ava": "^0.15.2",
+    "babel-preset-es2015-rollup": "^1.1.1",
+    "eslint": "^2.7.0",
+    "eslint-config-codeyellow": "^0.3.0",
+    "jquery": "^2.0.0",
+    "jsdom": "^9.2.1",
+    "rollup": "^0.29.0",
+    "rollup-plugin-babel": "^2.5.1"
+  },
+  "scripts": {
+    "lint": "eslint index.js && echo 'Linting succeeded.'",
+    "build": "rm -f dist/** && node build.js",
+    "test": "ava test.js",
```

Comment:
Could you use this?

``` 
"pretest": "npm run -s build && npm run -s lint",
"preversion": "npm run -s test",
```

This will make sure you can only deploy the latest dist files, and that the tests succeed. You can then also remove `npm run build` in the travis config.